.ci/scripts/run-behat

```diff
@@ -55,18 +30,5 @@ export BEHAT_PARAMS='{"extensions":{"Behat\\MinkExtension":{"base_url":"https://
 # Set Behat variables from environment variables
 export RELOCATED_WP_ADMIN=TRUE
```

Comment:
Could/Should this variable be set globally? It's here for the wp-cli commands that are called by Behat, right? Might there eventually be other process that need to call WP-CLI and use that variable?

Reply:
Perhaps, it would be trivial to set it in the Docker image.

Reply:
`RELOCATED_WP_ADMIN` is a property of the repository (how the code is organized); it therefore should not be set in the docker image, as that might be used with projects that do not organize their code in this way.